lib/Swagger/src/ContentEntity/Api/ContentToEntityRelationshipsApi.php

```diff
@@ -508,6 +520,11 @@ public function unrelateAllFromContentWithHttpInfo($content_id = null)
         } elseif (count($formParams) > 0) {
             $httpBody = $formParams; // for HTTP post (form)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)